tests/parser/ACTIONX.cpp

```diff
@@ -150,7 +152,7 @@ TSTEP
 
 BOOST_AUTO_TEST_CASE(TestActions) {
     Opm::SummaryState st(std::chrono::system_clock::now());
-    Opm::Action::Context context(st);
+    Opm::Action::Context context(st, WListManager());
```

Comment:
This constructor call is dangerous.  The reference data member in `Action::Context` does not extend the lifetime of the input, so anything that calls `Context::wlm()` and in any way **uses** the value returned from `wlm()` will get a dangling reference.

Reply:
Yes - the same with the SummaryState, I think that is OK.

Reply:
> Yes - the same with the `SummaryState`, I think that is OK.

I'm not sure we're talking about the same thing.  The `SummaryState` here is an actual object in the enclosing scope that lives until the end of that scope while the `WListManager` only exists for the duration of the constructor call.  Those are definitely not equivalent here.

Reply:
You are right - I misunderstood. Will fix